scrapers/GammaEntertainment.yml

```diff
@@ -28,17 +28,19 @@ sceneByURL:
       - devilsfilmparodies.com/
       - devilsgangbangs.com/
       - devonlee.com/
+      - disruptivefilms.com
```

Comment:
disruptivefilms.com has its own scraper that gets data directly from the api `Algolia_disruptivefilms.yml` and thus prefered over this one